tests/positive/files/units.go

```diff
@@ -320,3 +321,57 @@ func RemoveEnablementSymLinksforUnit() types.Test {
 		ConfigMinVersion: configMinVersion,
 	}
 }
+
+// testPresetsFileinAlphabeticalOrder checks if Ignition
+// writes the systemd presets in alphabeitcal order by
+// unit name into the 20-ignition.preset file.
+func testPresetsFileInAlphabeticalOrder() types.Test {
```

Comment:
You're right that this doesn't need to be exported (initial capital letter), but apparently that's what we've been doing (?), so let's do it here too for consistency.

Reply:
Thanks for catching this. I've updated the name.